lib/stripy.ex

```diff
@@ -49,15 +59,16 @@ defmodule Stripy do
       iex> Stripy.req(:post, "customers", %{"email" => "a@b.c", "metadata[user_id]" => 1})
       {:ok, %HTTPoison.Response{...}}
   """
-  def req(action, resource, data \\ %{}) when action in [:get, :post, :delete] do
+  def req(action, resource, data \\ %{}, opts \\ []) when action in [:get, :post, :delete] do
```

Comment:
Would you mind including an example inside the `@doc` definition that shows the usage of the 4th `opts` parameter? It will be really helpful when viewing the package in hexdocs.